rqt_shell/src/rqt_shell/shell.py

```diff
@@ -85,9 +86,16 @@ def __init__(self, context):
         super(Shell, self).__init__(context)
         self._context = context
         self.setObjectName('Shell')
+        self.args = self._parse_args(context.argv())
 
         self._widget = None
 
+    def _parse_args(self, argv):
+        parser = argparse.ArgumentParser(prog='rqt_shell', add_help=False)
+        parser.add_argument('-i', '--init_script', default=""
```

Comment:
I have changed it. But somehow, the invalid syntax is not popping up... strange. Maybe, my setup is invoking the pre-installed rqt_shell.